src/styles/blocks/catalog.scss

```diff
@@ -11,15 +14,22 @@
     justify-content: center;
     align-items: center;
     box-sizing: border-box;
-
     width: 200px;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: all $hoverTransitionDuration;
```

Comment:
Specify the transition property. Using `all` as a transition property is not the best approach